config/shields-io-production.yml

```diff
@@ -6,13 +6,20 @@ public:
       enabled: true
       url: https://metrics.shields.io/telegraf
       instanceIdFrom: env-var
-      instanceIdEnvVarName: HEROKU_DYNO_ID
+      instanceIdEnvVarName: FLY_ALLOC_ID
       envLabel: shields-production
 
   ssl:
-    isSecure: true
+    isSecure: false
 
   cors:
     allowedOrigin: ['http://shields.io', 'https://shields.io']
 
+  services:
+    gitlab:
+      authorizedOrigins: 'https://gitlab.com'
```

Comment:
I'm trying to remember why I needed to set this as an env var in the first place when we introduced support for GitLab API based data acquisition, but it's escaping my memory :shrug:

Reply:
I reckon what ended up as yaml config and what ended up as env vars was just us being a bit hit and miss over time tbh.

If you mean why we specifically needed to set this for GitLab, GitLab is the only service where we've got credentials set in production and there is no default `authorizedOrigins`. Everything else that takes an `authorizedOrigins` either has a default on `authorizedOrigins` or we allow credentials to be set for self-hosting purposes but don't do it ourselves.

Reply:
> If you mean why we specifically needed to set this for GitLab,

Sorry I forgot to resolve this thread last night. This was the question I had in mind but remembered that was the case